user-guide/install.md

```diff
@@ -41,90 +24,152 @@ The Ambassador Edge Stack is deployed to Kubernetes from the command line via ou
-
-Your hostname will appear in a pending state as the Ambassador Edge Stack configures automatic TLS. In the “Status” field, you will see the TLS status change. If you receive an error that your hostname does not qualify for ACME management, you can still configure TLS following [these instructions](../../reference/core/tls).
-
-To upgrade from evaluation mode, sign up for a free community license today from your Edge Policy Console.
-
-## Create Mappings
-
-Mappings allow you to associate parts of your domain with different URLs, IP addresses, or prefixes. Create your own Mappings in the administrative interface, the Edge Policy Console, to map out your own application.
-
-To show you how powerful the Ambassador Edge Stack is, follow the instructions to create a mapping to the site `httpbin.org`.
-
-**To do so:**
-
-1. Navigate to your Edge Policy Console.
-2. From the left menu, click the **Mappings** tab.
-3. On the right hand side, click the **(+)** button to add a mapping.
-4. In the “Mapping” field, enter a name that follows the naming conventions found [here](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names), such as `examplemapping`.
-5. To the right of the “Mapping” field, enter an existing namespace for your mapping, or leave it as `Default.`
-6. In the “Prefix URL” field, enter `/httpbin/`
-7. In the “Service” field, enter `httpbin.org:80`
-8. Click the **Save** button. Your new mapping will appear in the URL table below, and as a block in the Mappings section.
-9. To see how quickly the Mappings were applied, run the following command: `curl https://<hostname>/httpbin/` to print the HTML of the page.
-
-### Verify Mappings
-
-After you create Mappings, you can also verify that all of the CRDs were successfully installed using the command `kubectl get mappings`. `Kubectl` is the command-line tool that allows you to control Kubernetes.
-
-Because Ambassador is built on top of Kubernetes, it’s more than likely that you already have `kubectl` installed on your machine. However, if you don’t, you can install it following [these directions](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
-
-Kubernetes will print the Custom Resource Definitions that were installed with the Ambassador Edge Stack, and you’ll see that your new Mapping has been adding to the CRDs of the Ambassador Edge Stack.
+## Configure TLS for Automatic HTTPS
+
+If you have the ability to update your DNS, Ambassador can automatically configure a valid TLS certificate for you, eliminating the TLS warning. If you do not have the ability to update your DNS, skip to the next section.
+
+1. Update your DNS so that your domain points to the IP address for your cluster.
+
+2. In the Edge Policy Console, create a `Host` resource:
+   * Click on the **Hosts** tab.
+   * Click the **Add** button on the right.
+   * Enter your hostname (domain) in the hostname field.
+   * Check the "Use ACME to manage TLS" box.
```

Comment:
clickable things should be bolded. Areas of the UI that you are using for directional/location purposes should be in quotes. Ex: On the "Hosts" tab, click the **Add** button.